app/authentication/authenticator.py

```diff
@@ -1,5 +1,8 @@
 import logging
```

Comment:
I was going to wait for the pylint changes before sorting imports out as there will be lots of conflicts